readme.md

```diff
@@ -118,15 +149,22 @@ What is the difference between the two following lines of code?
 @artist.save
 @artist.save!
 ```
-
-> Your answer...
+The difference between the two is the 'bang' at the end of the last .save method.  It will throw an error if it does not save.  This is useful for the developer so that he/she will know something went wrong with saving the object rather than the method failing silently.
 
 ## Question 10
 
 Using jQuery, write an AJAX request to `http://tunr.com/artists` that would create a new artist with the name of 'Resin Laying Deer Figurine, Gold', and pop up a box saying "All done!" when complete.
 
 ```js
-// Your code starts here...
+
+var createArtist = $.ajax({
+    url: "http://tunr.com/artists",
+    method: "post",
+    data: JSON.stringify({name: "Resin Laying Deer Figurine", type: "Gold" }),
```

Comment:
dont believe you have to be extra explicit about the format of the data, i.e.
```
  data: {
    name: "Resin Laying Deer Figurine, Gold"
  }
```
should work as well, but like the your consideration